packages/dm-core/src/components/EntityView.tsx

```diff
@@ -28,13 +37,29 @@ export const EntityView = (props: IEntityView): React.ReactElement => {
     dimensions
   )
 
+  // Refresh Button stuff
+  const [reloadCounter, setReloadCounter] = useState(0)
+  const [mouseHoverOverRefreshButton, setMouseHoverOverRefreshButton] =
+    useState(false)
+  const [mouseCloseToRefreshButton, setMouseCloseToRefreshButton] =
+    useState(false)
```

Comment:
Cannot this hover effect be more elegantly be handled with CSS?
I think so... :)

Reply:
Hmm how so?

By wrapping it in a invisible container of 1/3 size and having the mouseEnter and mouseLeave property here?

Reply:
Could probably make the button opacity work with only css, but I don't think you can style different elements based on hovering another with pure CSS 🤔

Reply:
you can set different styling based on "hover" and other similar properties like so:
```
const ClickableIcon = styled.button`
  appearance: none;
  border: none;
  background-color: transparent;

  &:hover {
    color: gray;
    cursor: pointer;
  }
`
```